deltachat-ios/Controller/Settings/Proxy/ShareProxyViewController.swift

```diff
@@ -64,22 +67,14 @@ class ShareProxyViewController: UIViewController {
     required init?(coder: NSCoder) { fatalError() }
 
     private func setupConstraints() {
-
-        let qrImageRatio: CGFloat
-        if let image = qrContentView.image {
-            qrImageRatio = image.size.height / image.size.width
-        } else {
-            qrImageRatio = 1
-        }
-
         verticalCenterConstraint = contentStackView.centerYAnchor.constraint(equalTo: contentScrollView.centerYAnchor)
         contentTopAnchor = contentStackView.topAnchor.constraint(equalTo: contentScrollView.topAnchor, constant: 16)
         contentBottomAnchor = contentScrollView.bottomAnchor.constraint(equalTo: contentStackView.bottomAnchor, constant: 16)
 
         let constraints = [
-            qrContentView.widthAnchor.constraint(lessThanOrEqualTo: view.safeAreaLayoutGuide.widthAnchor, multiplier: 0.75),
+            qrContentView.widthAnchor.constraint(lessThanOrEqualTo: view.safeAreaLayoutGuide.widthAnchor, multiplier: 0.65),
```

Comment:
making it a little smaller, this is fine on all devices i tested.

however, having it more adaptive would be great!